Backendless/Persistence/BackendlessGeometryFactory.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using Weborb.Reader;
+using Weborb.Types;
+using Weborb.Util;
+
+namespace BackendlessAPI.Geo
+{
+  class BackendlessGeometryFactory : IArgumentObjectFactory
+  {
+    public Object createObject( IAdaptingType iAdaptingType)
+    {
+      if( iAdaptingType is NamedObject)
+        iAdaptingType = ((NamedObject) iAdaptingType).TypedObject;
+
+      if ( iAdaptingType.GetType() == typeof( NullType ) )
+        return null;
+
+      if ( iAdaptingType is AnonymousObject )
+      {
+        Dictionary<object, object> properties = ( Dictionary<object, object> )iAdaptingType.defaultAdapt();
+        String geoJson = ( String )properties["geoJson"];
+
+        if ( geoJson == null )
+          return null;
+
+        String geomClass = ( String )properties["geomClass"];
```

Comment:
no spaces around String. Should be (String) for casting. Space after ")":
` (String) properties["geomClass"]`